pkg/clients/clients.go

```diff
@@ -117,6 +117,11 @@ func (dcs *DynamicClientSet) gvkForKind(kind kinds.Kind) (*schema.GroupVersionKi
 	}
 
 	for _, gvResources := range resources {
+		// For some reason, the server is returning the old "extensions/v1beta1" GV before "apps/v1", so manually
+		// skip it.
+		if gvResources.GroupVersion == "extensions/v1beta1" {
+			continue
```

Comment:
I really do not like skipping the API versions here. If we have a preferred API version I think we should look for it and then fall back to this if we can't find it.